package.json

```diff
@@ -96,7 +96,7 @@
     "directory-tree-webpack-plugin": "^1.0.2",
     "duplexer": "^0.1.1",
     "eslint": "^7.8.1",
-    "eslint-loader": "^2.0.0",
+    "eslint-loader": "^4.0.2",
```

Comment:
I recommend to use https://github.com/webpack-contrib/eslint-webpack-plugin, `eslint-loader` was deprecated in near future

Reply:
Marked as resolved but eslint loader is still in here?